locked-nft/contracts/LockedNFT.cdc

```diff
@@ -0,0 +1,154 @@
+import NonFungibleToken from "./NonFungibleToken.cdc"
+
+pub contract LockedNFT {
```

Comment:
I think the contract name is not correct as in the literal sense it gives the vibe of having a type of NFT that itself going to produce NFT from a collection. So I think a better naming convention would be `NFTLocker`

Reply:
Changed to your suggestion.